### DISCUSSION TITLE
About English localization support

### DISCUSSION THREAD

**Discussion by dyphire** [dyphire/mpv-config#37] [Announcements]
English localization support has finally been completed and pushed, see https://github.com/dyphire/mpv-config/tree/eng

Note: It should be pointed out that the English branch is difficult to actively maintain. Because I personally do not use it, it increases maintenance costs. This requires other people who will use it in the future to spontaneously maintain and open PR, if any.

**Comment by Zabooby** [comment]
@dyphire I have translated input.conf but I can't be bothered to fork and make a pr. Could you review my changes and push to the English branch please? I can't attach a .conf file so just change the file extension: 
[input.txt](https://github.com/dyphire/mpv-config/files/15461296/input.txt)

**Reply by dyphire** [reply]
Thank you for what you do.

**Reply by Zabooby** [reply]
No problem. 

Are there any other files that affect the ui that can be translated. The only other line I can think of is [this](https://github.com/dyphire/mpv-config/blob/7a4c9be323f602c730db54aae68c9885f419db3f/script-opts/uosc.conf#L81) one in uosc.conf. 

Also could you add this line to line 201 in input.conf, I seem to have missed it. 
`F cycle-values sub-font "Noto Sans CJK SC" "Noto Sans CJK KR" "Noto Serif CJK SC" "Noto Serif CJK KR"; show-text "Subtitle font: ${sub-font}"  #menu: Subtitles > Switch Default Font
`

And finally, the profiles haven't been updated in mpv.conf yet.

**Reply by dyphire** [reply]
> Are there any other files that affect the ui that can be translated. The only other line I can think of is [this](https://github.com/dyphire/mpv-config/blob/7a4c9be323f602c730db54aae68c9885f419db3f/script-opts/uosc.conf#L81) one in uosc.conf.

Other than that there doesn't seem to be anything else that affects the UI.

> Also could you add this line to line 201 in input.conf, I seem to have missed it. `F cycle-values sub-font "Noto Sans CJK SC" "Noto Sans CJK KR" "Noto Serif CJK SC" "Noto Serif CJK KR"; show-text "Subtitle font: ${sub-font}" #menu: Subtitles > Switch Default Font `

It was just removed by me in the commit https://github.com/dyphire/mpv-config/commit/16a7d13173606629341a3e224a85eee077adbfbc, the item was used to switch between different Chinese fonts, not for English users.

> And finally, the profiles haven't been updated in mpv.conf yet.

I'll take a look at this later.

**Reply by Zabooby** [reply]
@dyphire Here is the updated controls line for uosc.conf:
`controls=menu,open-file,<idle>command:history:script-binding recentmenu/open?Recently Played,<idle>command:bookmarks:script-binding simplebookmark/open-list?Bookmarks Menu,<idle>command:file_copy:script-binding smartcopypaste_II/open-list?Clipboard Menu,gap,<!idle>command:analytics:script-binding stats/display-stats-toggle?Statistics,<stream>command:high_quality:script-binding quality_menu/video_formats_toggle?Streaming Quality,<audio,has_image>command:image:script-binding uosc/video#video?Cover,<has_many_edition>editions,<video>video,<has_audio,!audio>audio,<has_sub>subtitles,<has_chapter>chapters,space,<has_chapter>command:skip_previous:add chapter -1?Previous Chapter,<!idle,!image>speed,<has_chapter>command:skip_next:add chapter 1?Next Chapter,space,loop-file,shuffle,loop-playlist,gap,<has_playlist>prev,<has_playlist>playlist,<has_playlist>next,gap,audio-device,gap,fullscreen`
